data/simulations/FAH-Mpro-monomer-no-water.yml

```diff
@@ -0,0 +1,60 @@
+type: md
+title: SARS-CoV-2 main protease (apo, monomer) Folding@home simulations
+description: >
+  Folding@home projects 11743 and 11749 are made available here without solvent, but including salt.
+  Both of these projects were initiated from PDB structure `6lu7`, chain A (after removing the inhibitor (i.e. in apo form), as a monomer).
+  The split into two projects is simply the result of the architecture of Folding@home, there are no differences in the setups between these projects.
+  Please note the identical file names in both projects, there is no relation between the identically named files - all trajectories are initialized with different random velocities.
+  Each trajectory is available in the appropriate project folder with trajectory files formatted as `run<RUN>-clone<CLONE>.h5`,
+  with `<RUN>` indexing a set of trajectories started from the same conformation (i.e. all trajectories here are `RUN0`)
+  and `<CLONE>` denoting different replicates initiated from the same structure but different initial velocities.
+  The dataset contains 5688 trajectories - a subset of trajectories longer than 250 ns generated in these Folding@home projects. Frames are saved every 0.2 ns.
+
+  Chain A (i.e. a monomer of the protein, without the inhibitor or waters) was extracted from `6lu7` using PyMOL, and protonated and capped (ACE, NME) with Schrodinger's Maestro.
+  Simulations were performed in the NPT ensemble (310 K, 1 atm), in a cubic box with 1 nm padding, 150 mM NaCl, with hydrogen mass repartitioning (4 amu H mass), using `amber14SB` and `tip3p` forcefields.
+  OpenMM 7.4.1 was used. System was equilibrated for 5 ns using 2 fs timestep with default OpenMM Langevin integrator,
+  then for 1.25 ns using 4 fs timestep with OpenMMTools custom Langevin integrator using `V R O R V` splitting.
+  All Folding@home trajectories were then seeded with random velocities from this system.
+
+  The dataset is available through both the Open Science Framework and AWS Public Datasets. Links are provided to GUIs for both below.
+  Additionally, the AWS dataset can be accessed through the command line interface (CLI) as follows:
+    1) Install the AWS CLI: `conda install awscli` or `pip install awscli`
```

Comment:
If you need to install the [AWS CLI](), you can use [conda]():
```bash
conda install awscli
```
or `pip`
```
pip install awscli
```
...

Reply:
Can you change this to `conda install -c conda-forge awscli`, in case folks don't have conda-forge added?